agenta-cli/agenta/sdk/agenta_init.py

```diff
@@ -171,7 +171,7 @@ def pull(self, config_name: str = "default", environment_name: str = None):
                     + str(ex)
                 )
         try:
-            self.set(**config["parameters"])
+            self.set(**config.parameters)
```

Comment:
I guess this was the main source of the bug in this case. It was creating this exception, no?

Reply:
Yes, yes. It was.